lib/commands/new.js

```diff
@@ -49,7 +46,8 @@ module.exports = function(args, options, callback, ee) {
 
   // 2. Find out what the user wants to do
   function prompt(cb) {
-    inquirer.prompt(util.questions(options), function(answers) {
+    inquirer.prompt(util.questions(options)).then( function(answers) {
+      console.dir(answers)
```

Comment:
Not sure why this is here. Unrelated to the PR title.

Reply:
Latest commit removes the `console.dir`, but the change to a promise is because of updated version.